app/api/helpers/system_notifications.py

```diff
@@ -23,16 +24,255 @@
+    """
+    accept_event_role_action = NotificationAction(
+        subject='event-role',
+        subject_id=event_id,
+        link=invitation_link,
+        action_type='view'
+    )
+    save_to_db(accept_event_role_action)
+    return [accept_event_role_action]
+
+
+def get_new_session_notification_actions(session_id, link):
+    """
+    Get the actions associated with a notification of an event getting a new session proposal.
+    :param session_id: id of the session.
+    :param link: link to view the session.
+    :return: actions
+    """
+    view_session_action = NotificationAction(
+        subject='session',
+        link=link,
+        subject_id=session_id,
+        action_type='view'
+    )
+    save_to_db(view_session_action)
+    return [view_session_action]
+
+
+def get_session_schedule_notification_actions(session_id, link):
+    """
+    Get the actions associated with a notification of change in schedule of a session.
+    :param session_id: id of the session.
+    :param link: link to view the session.
+    :return: actions
+    """
+    view_session_action = NotificationAction(
+        subject='session',
+        link=link,
+        subject_id=session_id,
+        action_type='view'
+    )
+    save_to_db(view_session_action)
+    return [view_session_action]
+
+
+def get_next_event_notification_actions(event_id, link):
+    """
+    Get the actions associated with a notification of next event.
+    :param event_id: id of the event.
+    :param link: link to view the event.
+    :return: actions
+    """
+    view_event_action = NotificationAction(
+        subject='event',
+        link=link,
+        subject_id=event_id,
+        action_type='view'
+    )
+    save_to_db(view_event_action)
+    return [view_event_action]
+
+
+def get_session_accept_reject_notification_actions(session_id, link):
+    """
+    Get the actions associated with a notification of a session getting accepted/rejected.
+    :param session_id: id of the session.
+    :param link: link to view the session.
+    :return: actions
+    """
+    view_session_action = NotificationAction(
+        subject='session',
+        link=link,
+        subject_id=session_id,
+        action_type='view'
+    )
+    save_to_db(view_session_action)
+    return [view_session_action]
+
+
+def get_invite_papers_notification_actions(cfs_link, submit_link):
+    """
+    Get the actions associated with an invite to submit papers.
+    :param cfs_link: link of call for speakers.
+    :param submit_link: link to submit papers.
+    :return:
+    """
+    view_cfs_action = NotificationAction(
+        subject='call-for-speakers',
+        link=cfs_link,
+        action_type='view'
+    )
+    submit_paper_action = NotificationAction(
+        subject='call-for-speakers',
+        link=submit_link,
+        action_type='submit'
+    )
+    save_to_db(view_cfs_action)
+    save_to_db(submit_paper_action)
+    return [view_cfs_action, submit_paper_action]
```

Comment:
You can simply use `return view_cfs_action, submit_paper_action` instead of returning lists.

Reply:
This will be a tuple right ? I want it to be a list.